sycl/test/regression/vec_value_type.cpp

```diff
@@ -0,0 +1,11 @@
+// RUN: %clangxx -fsycl %s
+#include <sycl/sycl.hpp>
+
+using namespace std;
```

Comment:
std include are missing (e.g. `#include <type_traits>`)

Reply:
They must be included under sycl.hpp as this compiles fine locally.

Reply:
You use `std::is_same` and SYCL spec doesn't guarantee that sycl.hpp must provide this template. 
You must explicitly include headers with declarations you use and do not assume that DPC++ implementation will always implicitly include `<type_traits>` header, even though it does so today.